account/manager_test.go

```diff
@@ -1152,6 +1221,65 @@ func TestAccountDeposit(t *testing.T) {
 			version:    VersionInitialNoVersion,
 			newVersion: VersionTaprootEnabled,
 		},
+		{
+			name: "happy path version 2",
+			fundedOutputAmount: depositAmount +
+				accountInputFeesTaproot,
+
+			// We'll use the lowest fee rate possible, which should
+			// yield a transaction fee of 304 satoshis when taking
```

Comment:
nit: I think that specifics like "304" will become stale as they are hard to keep up-to-date. A assertion in code would be an alternative. I would suggest removal from the comment.

Reply:
Yeah, good call. I replaced the magic number with a constant and removed the comments.